src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/EnvelopeCreator.java

```diff
@@ -93,6 +96,21 @@ public static Envelope envelope(String jurisdiction, Status status) {
         return envelope;
     }
 
+    public static InputScannableItem inputScannableItem(InputDocumentType docType) {
+        return new InputScannableItem(
```

Comment:
It would be best to make every field have a different value. This would help cover cases where values are assigned to wrong fields.

Reply:
done